src/test/scala/examples/BoreSpec.scala

```diff
@@ -0,0 +1,36 @@
+// See README.md for license details.
+
+package examples
+
+import chisel3._
+import chisel3.iotesters.PeekPokeTester
+import org.scalatest.{FreeSpec, Matchers}
+
+class Constant extends MultiIOModule {
+  val x = Reg(UInt(6.W))
+  x := 42.U
+}
+
+class Expect extends MultiIOModule {
+  val y = IO(Output(UInt(6.W)))
+  y := 0.U
+}
+
+class BoreTop extends MultiIOModule {
+  val y = IO(Output(UInt(6.W)))
+  val constant = Module(new Constant)
+  val expect = Module(new Expect)
+  y := expect.y
+
+  util.experimental.BoringUtils.bore(constant.x, Seq(expect.y))
+}
+
+class BoreSpec extends FreeSpec with Matchers {
+  "Boring utils should work in io.testers" in {
+    iotesters.Driver(() => new BoreTop) { c =>
+      new PeekPokeTester(c) {
+        expect(c.y, expected = 42)
+      }
+    } should be(true)
```

Comment:
When this fails, it fails with:
```
[info] - Boring utils should work in io.testers *** FAILED ***
[info]   false was not true (BoreSpec.scala:30)
```
I realize it's beyond the purpose of this PR, but it would be nice if there was a more explicit error message.